jujugui/static/gui/src/app/components/search-results/search-results.js

```diff
@@ -18,18 +18,115 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('search-results', function() {
+YUI.add('search-results', function(Y) {
+
+  var views = Y.namespace('juju.views'),
+      Templates = views.Templates;
 
   juju.components.SearchResults = React.createClass({
+    search: function() {
+      /*
+       * TODO: replace this placeholder and its dummy data with live call to
```

Comment:
This is trifling, but can you make this an XXX comment instead? Most of canonical is familiar with looking for those.